{"uuid":"a858eb04-abc0-42c2-8e07-0cb10b6797d2","historyId":"d913498d84ff1448d80fb8f49e4e8e2","fullName":"secondExercise.loginWithInvalidCredentials","labels":[{"name":"package","value":"secondExercise"},{"name":"testClass","value":"secondExercise"},{"name":"testMethod","value":"loginWithInvalidCredentials"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"FirstSeleniumProject"},{"name":"subSuite","value":"secondExercise"},{"name":"host","value":"DESKTOP-950GMBS"},{"name":"thread","value":"11128@DESKTOP-950GMBS.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"loginWithInvalidCredentials","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Cannot invoke \"org.testng.asserts.SoftAssert.assertAll()\" because \"this.softAssertions\" is null","trace":"java.lang.NullPointerException: Cannot invoke \"org.testng.asserts.SoftAssert.assertAll()\" because \"this.softAssertions\" is null\r\n\tat secondExercise.loginWithInvalidCredentials(secondExercise.java:37)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:829)\r\n\tat org.testng.TestRunner.run(TestRunner.java:602)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:330)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1176)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1099)\r\n\tat org.testng.TestNG.run(TestNG.java:1067)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","description":"login with invalid credentials","steps":[{"name":"Enter Username","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"username","value":"locked_out_user"}],"start":1705142977434,"stop":1705142977862},{"name":"Enter password","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"password","value":"secret_sauce"}],"start":1705142977863,"stop":1705142977997},{"name":"Click login buton","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1705142977998,"stop":1705142978103}],"attachments":[],"parameters":[],"start":1705142976480,"stop":1705142981384}